functions/go/apply-setters/README.md

```diff
@@ -4,12 +4,29 @@
 
 <!--mdtogo:Short-->
 
-Apply setter values on resource fields. Setters serve as parameters for template-free
-setting of field values.
+Update the field values parameterized by setters.
```

Comment:
What's a "setter"? You're defining setters in terms of setters. See the problem?

Reply:
This line describes what the function does. As a user I read this line to know what this function does, if I don't know what setters mean, I just look down and see definition of setters.